CMake/External_OpenCV.cmake

```diff
@@ -21,6 +24,23 @@ else()
   set(fletch_ENABLE_OpenCV_FFmpeg CACHE INTERNAL FALSE)
 endif()
 
+# Allow OpenCV's VTK option to be explicitly turned off
+if(fletch_ENABLE_VTK)
+  # By default turn opencv-VTK support off, even when building with VTK
+  option(fletch_ENABLE_OpenCV_VTK "Build OpenCV with VTK support" FALSE )
+  mark_as_advanced(fletch_ENABLE_OpenCV_VTK)
+else()
+  set(fletch_ENABLE_OpenCV_VTK CACHE INTERNAL FALSE)
```

Comment:
Change the ALL the set calls with INTERNAL to an unset call
such as:
unset(fletch_ENABLE_OpenCV_VTK CACHE)

https://github.com/Kitware/fletch/pull/166/commits/083780f00d8c1c77d9bb329a85d3c594fa26d3c0